core/src/main/java/jenkins/model/FingerprintFacet.java

```diff
@@ -103,6 +108,27 @@ public long getTimestamp() {
         return timestamp;
     }
 
+    /**
+     * Returns whether Fingerprint deletion has been blocked by this Facet.
+     */
+    public boolean isFingerprintDeletionBlocked() {
+        return fingerprintDeletionBlocked;
+    }
+
+    /**
+     * Blocks deletion of Fingerprint associated with this Facet.
+     */
+    public void blockFingerprintDeletion() {
```

Comment:
Will the tools that discover getters and setters (like Pipeline, Configuration as Code, etc.) recognize the prefix "block" and the prefix "unblock" as setters?  Are any of them ever expected to need to discover these?

Reply:
IIUC, the plugins do need to be able to discover these setters (since they are the ones that create the facets).

Reply:
I'm with Mark here, better to use well-known names for setters and getter

Reply:
Hi, I had misunderstood. I am happy to convert it to the Jenkins standards. It would be really helpful if you could point me to a code snippet which uses these standards.

Reply:
It would be a way to go (standard Java setter implemetation)

```
    public void setFingerprintDeletionBlocked(boolean value) {
```